package.json

```
@@ -26,6 +26,7 @@
 		"pretest": "npm run npmcopy && npm run build && npm run lint",
 		"start": "npm run npmcopy && node build/tasks/build-watch.js",
 		"test:browser": "npm run pretest && npm run test:unit -- -b chrome -b firefox -h",
+		"test:ie": "npm run pretest && npm run test:unit -- -v -b ie",
```

```suggestion
		"test:ie": "npm run pretest && npm run test:unit -- -b ie -h",
```

Because this is Edge, I think this can be headless.